backend/api/Areas/Persons/Models/Person/OrganizationLinkModel.cs

```diff
@@ -0,0 +1,22 @@
+
+namespace Pims.Api.Areas.Persons.Models.Person
+{
+    /// <summary>
+    /// Provides a contact-oriented organization model.
+    /// This model contains a sub-set of organization fields as it is only used to link a person to an organization.
+    /// </summary>
+    public class OrganizationLinkModel
```

Comment:
This class looks like is formating the organization to be rendered already. Why not pass the organization instead and let the UI handle how it is rendered?

Reply:
This is to match the results from the auto-complete for organizations. We could improve the auto-complete to return the whole organization but was a lot of work and didn't want to refactor the ADD form as well

Reply:
I see, we should standardize those too

Reply:
we should